modules/virt-about-resource-quota-limits.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-specifying-nodes-for-vms.adoc
+
+:_content-type: CONCEPT
+[id="virt-about-resource-quota-limits_{context}"]
+= About resource limits
+
+You can only set `ResourceQuotas` for virtual machines if the resource quota has a set of requests. If the resource quota contains both requests and limits, the resource limits on the virtual machine must be set.
```

Comment:
- I am confused by this intro, as well as its location in this assembly (which doesn't mention resource quotas).

- I think this should be a procedure rather than a concept. 

- After reviewing the bug, I would reword the intro to something like this:
> Resource quotas that only use requests automatically work with virtual machines (VMs). If your resource quota uses limits, you must manually set resource limits on VMs. Resource limits must be at least 100 MiB larger than resource requests.
- Re: s/megabytes/MiB: the `Mi` in the YAML makes me think that it is supposed to be MiB (mebibytes). The ISG also seems to recommend just writing `MiB` instead of spelling it out: https://www.ibm.com/docs/en/ibm-style?topic=measurement-units

- I think there should be a procedure step or two; something like this:

```
. Set limits for a VM by editing the `VirtualMachine` manifest. For example:
+
<yaml example>

. <step to apply changes>
```

Reply:
Pan, +1 on your suggested wording.

Reply:
Thanks, Pan. Much clearer.